lab_1_keywords_tfidf/main.py

```diff
@@ -48,7 +66,19 @@ def calculate_frequencies(tokens: list[str]) -> Optional[dict[str, int]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(tokens, list) and len(tokens) != 0:
```

Comment:
reverse the statement and remove `None` immediately

Reply:
done